nexus-repository-helm/src/main/java/org/sonatype/repository/helm/internal/util/ProvenanceParser.java

```diff
@@ -0,0 +1,58 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2018-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.repository.helm.internal.util;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+
+import javax.inject.Named;
+import javax.inject.Singleton;
+
+import org.sonatype.repository.helm.HelmAttributes;
+import org.sonatype.repository.helm.internal.database.HelmProperties;
+
+@Named
+@Singleton
+public class ProvenanceParser
```

Comment:
Maybe it is possible to parse it same as in YamlParser with YAMLFactory? I see that prov file is quite simple to prov